asciidoc/courses/genai-fundamentals/modules/3-knowledge-graphs/lessons/1-what-is-a-knowledge-graph/lesson.adoc

```diff
@@ -5,7 +5,7 @@
 
 [.slide.discrete]
 == Knowledge Graphs
-[quote]
+
```

Comment:
I think this should be a quote. Its a direct quote from us at Neo4j. There is no definition for a knowledge Graph

Reply:
I removed it for consistency reasons, it looks like an abstract. But maybe this is the best way to emphasize?

Reply:
>best way to emphasize?

I think so, it formats it differently in GA too.